composer.json

```diff
@@ -1,5 +1,5 @@
 {
-  "name": "moxie-leean/wp-endpoints-archive",
+  "name": "moxie-lean/wp-endpoints-archive",
   "description": "wp-endpoints-archive",
```

Comment:
Basically the main thing with the endpoints is to make it agnostic from WP Since are going to be used from a FE that usually does not know nothing about WP, archive makes sense in terms of WP outside of a WP world not su much, but if you have any other suggestions we can discuss about the naming.

Thanks.

> On Mar 31, 2016, at 5:19 AM, Caridad Zapatero Contreras <notifications@github.com> wrote:
> 
> In composer.json <https://github.com/moxie-leean/wp-endpoints-collection/pull/2#discussion_r58037571>:
> 
> > @@ -1,5 +1,5 @@
> >  {
> > -  "name": "moxie-leean/wp-endpoints-archive",
> > +  "name": "moxie-lean/wp-endpoints-archive",
> >    "description": "wp-endpoints-archive",
> Why the different naming (Collection vs Archive)?
> 
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub <https://github.com/moxie-leean/wp-endpoints-collection/pull/2/files/86356df199c5bac0bd0cfbda93ff685a610a4205#r58037571>